apps/condo/domains/notification/adapters/apple/AppleMessaging.js

```diff
@@ -127,7 +129,14 @@ class AppleMessaging {
 
             logger.info({ msg: 'sendPush before request', headers, options, payload })
 
-            const stream = this.#session.request(headers)
+            let stream
+            for (let retryCounter = 0; retryCounter < 5; retryCounter++) {
```

Comment:
Looks like this code will send 5 push messages.
Every loop iteration you rewrite the `stream` variable. After the loop you have another "onResponse" callback. Which stream this callback is?

You need to wait the answer for the first request. If the answer is erroneous you should repeat the request.